src/app/Http/Requests/Admin/BouncerAssignAbilityRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Requests\Admin;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class BouncerAssignAbilityRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    public function rules(): array
+    {
+        return [
+            'role_name'  => ['required',  Rule::unique('roles', 'name')],
```

Comment:
'role_name' is not unique, we are not creating a role. Shoud be `Rule::exists('role', 'name')`